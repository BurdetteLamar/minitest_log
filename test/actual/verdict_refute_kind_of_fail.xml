<log>
  <summary_ verdicts='3' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_refute_kind_of?' type='fail'>
    <section_ name='Args=[Exception, #&lt;ArgumentError: ArgumentError&gt;, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_refute_kind_of?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='Class' value='Exception'/>
        <actual_ class='ArgumentError' value='#&lt;ArgumentError: ArgumentError&gt;'/>
        <exception_ class='Minitest::Assertion' message='Expected #&lt;ArgumentError: ArgumentError&gt; to not be a kind of Exception.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[Numeric, 0, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_refute_kind_of?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Class' value='Numeric'/>
        <actual_ class='Integer' value='0'/>
        <exception_ class='Minitest::Assertion' message='Expected 0 to not be a kind of Numeric.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
