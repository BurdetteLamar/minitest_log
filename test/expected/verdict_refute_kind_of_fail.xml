<log>
  <summary_ verdicts='2' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_refute_kind_of?' type='fail'>
    <section_ name='Args=[Exception, #&lt;ArgumentError: ArgumentError&gt;]'>
      <verdict_ method='verdict_refute_kind_of?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='Class' value='Exception'/>
        <actual_ class='ArgumentError' value='#&lt;ArgumentError: ArgumentError&gt;'/>
        <exception_ class='Minitest::Assertion' message='Expected #&lt;ArgumentError: ArgumentError&gt; to not be a kind of Exception.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[Numeric, 0]'>
      <verdict_ method='verdict_refute_kind_of?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Class' value='Numeric'/>
        <actual_ class='Integer' value='0'/>
        <exception_ class='Minitest::Assertion' message='Expected 0 to not be a kind of Numeric.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
  </section_>
</log>
